project/Base.scala

```diff
@@ -41,7 +41,7 @@ class Base extends Build {
     // XXX
     //conflictManager := ConflictManager.strict,
     resolvers ++= Seq(
-      "twitter-repo" at "https://maven.twttr.com",
+      // "twitter-repo" at "https://maven.twttr.com",
```

Comment:
maybe add a comment reminding the reader what this comment should remind them of